resources-domain/packages/resources-search-config/index.ts

```diff
@@ -19,7 +19,29 @@ import { convertIndexDocument } from './convertIndexDocument';
 import { getCreateIndexParams } from './getCreateIndexParams';
 
 export const resourceSearchConfiguration: SearchConfiguration = {
```

Comment:
This is for KHP or is a default configuration?

Reply:
For KHP - although I'm not sure how much we will customise it from account to account